src/main/backend.ts

```diff
@@ -188,7 +188,7 @@ export class MainBackend extends BaseBackend<ElectronOptions> implements CommonB
     // error.
     const dsnString = this._options.dsn;
     if (!dsnString) {
-      throw new SentryError('Invariant exception: install() must not be called when disabled');
```

Comment:
I modified this error message because I hit it in testing when not passing a DSN and it didn't point me in the right direction.